tests/framework/validators/BooleanValidatorTest.php

```
@@ -70,24 +76,43 @@ public function testValidateAttributeAndError(): void
 
     public function testErrorMessage(): void
     {
+        $this->mockWebApplication();
         $validator = new BooleanValidator([
             'trueValue' => true,
             'falseValue' => false,
             'strict' => true,
         ]);
         $validator->validate('someIncorrectValue', $errorMessage);
```

PHPMD (2.15.0)

[warning] Avoid using undefined variables such as '$errorMessage' which will lead to PHP notices. (undefined)

(UndefinedVariable)